src/main/java/utils/SeleniumDriver.java

```diff
@@ -32,6 +32,7 @@
         ChromeOptions chromeOptions = new ChromeOptions();
         chromeOptions.addArguments("--no-sandbox");
         chromeOptions.addArguments("--headless");
+        chromeOptions.addArguments("window-size=1920x1040");
```

Comment:
It should be `--window-size`, also can we put it into config?

Reply:
In Java it works that way-https://www.tabnine.com/code/java/methods/org.openqa.selenium.chrome.ChromeOptions/addArguments

No need to put it config as this will be base class and it doesn't matter if we are running in headless or browser this can be constant.

Reply:
According to the [javadoc](https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/chromium/ChromiumOptions.html#addArguments(java.util.List)), both are working, so it's better to use consistent style. 
I understand that it can be same for headless or not, but we may want to change it at some point going forward, so having it configurable would be a plus